### ISSUE TITLE
Correct accessibility issues with schemas

### ISSUE THREAD

**Issue by glowcloud** [swagger-api/swagger-ui#10705]
- [ ] focus is lost when opening or closing the schema in schemas section

- [ ] OpenAPI 2.0/3.0: the information about the property being expanded is narrated only after navigating into […]/information group

https://github.com/user-attachments/assets/19213d46-80e3-4892-ba70-fecad2090bef

- [ ] OpenAPI 3.1: keyboard navigation gets trapped in expanded properties, jumps between the property name and collapse all button

https://github.com/user-attachments/assets/242dde3a-143d-4480-96f3-d48e7287e3d5

- [ ] OpenAPI 3.1: focus is not visible for schema name and properties with tab navigation

- [ ] OpenAPI 3.1: no name/label for article section 

- [ ] OpenAPI 2.0/3.0 schema buttons - provide indication which property the button expands/collapses 

- [ ] OpenAPI 3.1 schema buttons - provide description for expanded/collapsed state 

- [ ] required property indicated by `*` - provide label/description `required` for screen reader users instead of `*` 

This task covers issues originally reported in:
- https://github.com/swagger-api/swagger-ui/issues/5248
- https://github.com/swagger-api/swagger-ui/issues/7350
